docs/autonomy.md

```diff
@@ -24,21 +24,24 @@ Usage: autonomy deploy build [OPTIONS] [KEYS_FILE]
   Build deployment setup for n agents.
 
 Options:
-  --o PATH                  Path to output dir.
-  --n INTEGER               Number of agents.
-  --docker                  Use docker as a backend.
-  --kubernetes              Use kubernetes as a backend.
-  --dev                     Create development environment.
-  --version TEXT            Specify deployment version.
-  --force                   Remove existing build and overwrite with new one.
-  --packages-dir PATH       Path to packages dir (Use with dev mode)
-  --open-aea-dir PATH       Path to open-aea repo (Use with dev mode)
-  --open-autonomy-dir PATH  Path to open-autonomy repo (Use with dev mode)
-  --remote                  To use a remote registry.
-  --local                   To use a local registry.
-  -p                        Ask for password interactively
-  --password PASSWORD       Set password for key encryption/decryption
-  --help                    Show this message and exit.
+  --o PATH                        Path to output dir.
+  --n INTEGER                     Number of agents.
+  --docker                        Use docker as a backend.
+  --kubernetes                    Use kubernetes as a backend.
+  --dev                           Create development environment.
+  --force                         Remove existing build and overwrite with new
+                                  one.
+  --log-level [INFO|DEBUG|WARNING|ERROR|CRITICAL]
+                                  Logging level for runtime.
+  --packages-dir PATH             Path to packages dir (Use with dev mode)
+  --open-aea-dir PATH             Path to open-aea repo (Use with dev mode)
+  --open-autonomy-dir PATH        Path to open-autonomy repo (Use with dev
+                                  mode)
+  --remote                        To use a remote registry.
+  --local                         To use a local registry.
+  -p                              Ask for password interactively
+  --password PASSWORD             Set password for key encryption/decryption
+  --help                          Show this message and exit.
```

Comment:
These options are subject to change overtime. Maybe open a ticket to add a test for them.